File: Day-14/Pattern19.py
n = int(input())
def PatternNineteen(n):
    i = 1
    while (i <=n):
        space = n - i
        while (space):
            print(' ', end='')
            space -= 1
        
        star = 1
        while(star <= i):
            print('*',end='')
            star +=1
        
        i+=1
        print(end='\n')       
PatternNineteen(n)

File: Day-15/Pattern19.js
function PatternNineteen(n) {
  let i = 1;
  while (i <= n) {
    let str = "";
    let space = n - i;

    while (space) {
      str += " ";
      space--;
    }

    let star = i;
    while (star) {
      str += "*";
      star--;
    }

    console.log(str);
    i++;
  }
}

PatternNineteen(4);
